corehq/apps/cloudcare/templates/formplayer/case_detail.html

```diff
@@ -10,19 +10,29 @@
     <td><%- data %></td>
 </script>
 
+<script type="text/template" id="detail-view-tab-list-template">
+    <ul class="nav nav-pills">
```

Comment:
i think the right css for tabs would be b3 tabs here: http://getbootstrap.com/javascript/#tabs